virt/release_notes/virt-4-20-release-notes.adoc

```diff
@@ -255,6 +256,9 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 * {VirtProductName} links a service account token in use by a pod to that specific pod. {VirtProductName} implements a service account volume by creating a disk image that contains a token. If you migrate a VM, then the service account volume becomes invalid. (link:https://issues.redhat.com/browse/CNV-33835[*CNV-33835*])
 ** As a workaround, use user accounts rather than service accounts because user account tokens are not bound to a specific pod.
 
+//CNV-70798
+* When upgrading from {VirtProductName} 4.19 to 4.20, you must remove the `wasp-agent` component before initating the upgrade. For more information about removing the `wasp-agent` component, see xref:../../virt/post_installation_configuration/virt-configuring-higher-vm-workload-density.adoc#virt-removing-wasp-agent_virt-configuring-higher-vm-workload-density[Removing the wasp-agent component].
```

Comment:
It is unusual that this known issue doesn't link to a bug or mention why you need to remove the `wasp-agent` component. (I was also confused about why this is in the virtualization section instead of install/update, but since the bug is presumably with wasp-agent, that makes sense.)